### PR TITLE
create task_12

### PR DESCRIPTION
**Pull request by TanyaLevchenya**

### CODE REVIEW

File: task_12/grid.html [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+                </div>
+            </div>
+
+            <div class="menu-item__price indents">
+                <div>14 р</div>
+                <div>740 г</div>
+            </div>
+        </div>
+
+        <div class="menu-item__popup__window">
```
Comment (Killerkip10) [inline task_12/grid.html:97]: Incorrect name
_menu-item__popup-window_ is correct

File: task_12/style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    position: absolute;
+    top: -200px;
+    background-color: #84bd2b;
+    color: white;
+    padding: 10px;
+    text-align: left;
+}
+.menu-item__desc:hover ~ .menu-item__popup__window{
+    top: 0;
+    transition: 0.5s;
```
Comment (Killerkip10) [inline task_12/style.css:32]: Please don't use transition inside :hover move to .menu-item__popup__window

**Review by Killerkip10 — COMMENTED** [review]
## Task 12
-  (1.6) Animation is too sharp. (-5 points);
### Total 95 points

File: task_12/grid.html (hunk 1 above)
Comment (TanyaLevchenya) [reply task_12/grid.html:97]: correct

**Pull request merged**